pages/eventos/editar/[id].tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react'
+import { NextPage } from 'next'
+import { useRouter } from 'next/router'
+import { useMutation, useLazyQuery } from '@apollo/client'
+import { GET_EVENT_BY_ID } from '../../../services/GraphQL/queries/events'
+import { UPDATE_EVENT } from '../../../services/GraphQL/mutations/events'
+import { EventBase } from '../../../services/GraphQL/types/events'
+import ClientOnly from '../../../views/Shared/ClientOnly'
+import styles from '../../../styles/Home.module.css'
+import { formatDate } from '../../../services/utils/dateFormat'
+
+const EditEvent: NextPage = () => {
+  const [getOriginalEventData, { loading, data }] = useLazyQuery(GET_EVENT_BY_ID)
+  const { push, query } = useRouter()
+  const [updatedEvent, setUpdatedEvent] = useState<EventBase>({
+    title: data?.title || '',
+    description: data?.description || '',
+    date: data?.date || '',
+    type: data?.type || '',
+  })
+  const [date, setDate] = useState('')
+  const [time, setTime] = useState('')
+
+  const [updateEvent] = useMutation(UPDATE_EVENT)
+
+  useEffect(() => {
+    const getCurrentData = async (id: string) => {
+      const { called, error } = await getOriginalEventData({
+        variables: {
+          id,
+        },
+      })
+
+      if (error || !called) {
+        console.error(error)
+      }
```

Comment:
not should return something or thrown a error?

Reply:
I think in the long run we should use the `frontendErrors` table in Hasura, but have to create an issue to change all error calls to send the errors to that database in production. So in development we can console the error, but in production we send to database